src/nvim/usercmd.c

```diff
@@ -1789,6 +1789,14 @@ Dict commands_array(buf_T *buf, Arena *arena)
     PUT_C(d, "keepscript", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_KEEPSCRIPT)));
     PUT_C(d, "preview", BOOLEAN_OBJ(!!(cmd->uc_argt & EX_PREVIEW)));
 
+    if (cmd->uc_preview_luaref != LUA_NOREF) {
+      PUT_C(d, "preview_callback", LUAREF_OBJ(api_new_luaref(cmd->uc_preview_luaref)));
```

Comment:
Doesn't `nvim_create_user_command` call this `preview`? Should always use the same names for input/output if possible. Ideally, they should be "roundtrippable", but even if not..